libraries/botbuilder/tests/cloudAdapter.test.js

```diff
@@ -87,6 +87,29 @@ describe('CloudAdapter', function () {
 
             mock.verify();
         });
+
+        it('calls processActivityDirect', async function () {
+            const logic = async (context) => {
+                context.turnState.set(INVOKE_RESPONSE_KEY, {
+                    type: ActivityTypes.InvokeResponse,
+                    value: {
+                        status: 200,
+                        body: 'invokeResponse',
+                    },
+                });
+            };
+
+            const mock = sandbox.mock(adapter);
+            mock.expects('processActivity')
+                .withArgs('Bearer Authorization', { type: 'invoke', value: 'invoke' }, logic)
```

Comment:
I would declare the activity at the beginning and re-use it later. This way, we don't have duplicated code, and it's also clearer that we are passing an activity as a parameter to the methods.